### ISSUE TITLE
Help the JUG BB to improve the hosting of their website

### ISSUE THREAD

**Issue by obfischer** [hackergarten-berlin/organizer#7]
The Java Usergroup Berlin-Brandenburg hosts their homepage on the private hosting account of @obfischer. This has some serious consequenes.

For example Oliver is the only one who can maintain the site.

It would be great to work on a hosting solution which allows the Java Usergroup to enable various people to maintain the site.

**Comment by ehmkah** [comment]
Good idea. Which technology is used for creating the content? Do you think of an automated process which deployes the page after changes have been made and were approved? Something similar to https://pages.github.com/ 

The webpage can be found here  http://www.jug-berlin-brandenburg.de

**Comment by obfischer** [comment]
Currently I use JBake. The page is build _on my machine_ and deployed via `rsync` to my account at Hetzner.
